recipes/sfepy/run_test.py

```diff
@@ -0,0 +1 @@
+import sfepy
```

Comment:
Please delete this file and just add a `test:import` section to the main recipe.